Source/Scene/Label.js

```diff
@@ -701,6 +703,27 @@ define([
                     }
                 }
             }
+        },
+
+        _clusterRender : {
```

Comment:
This doesn't need an underscore; it can just be `@private`.  Not sure that I would use "render" here though since we never use in this context.  Perhaps just `clusterShow`.